vital/algo/video_input.cxx

```diff
@@ -8,22 +8,41 @@
  */
 
 #include <vital/algo/video_input.h>
+
 #include <vital/algo/algorithm.txx>
 
 namespace kwiver {
+
 namespace vital {
+
 namespace algo {
 
 // ------------------------------------------------------------------
-const algorithm_capabilities::capability_name_t video_input::HAS_EOV( "has-eov" );
-const algorithm_capabilities::capability_name_t video_input::HAS_FRAME_NUMBERS( "has-frame-numbers" );
-const algorithm_capabilities::capability_name_t video_input::HAS_FRAME_TIME( "has-frame-time" );
-const algorithm_capabilities::capability_name_t video_input::HAS_FRAME_DATA( "has-frame-data" );
-const algorithm_capabilities::capability_name_t video_input::HAS_FRAME_RATE( "has-frame-rate" );
-const algorithm_capabilities::capability_name_t video_input::HAS_ABSOLUTE_FRAME_TIME( "has-abs-frame-time" );
-const algorithm_capabilities::capability_name_t video_input::HAS_METADATA( "has-metadata" );
-const algorithm_capabilities::capability_name_t video_input::HAS_TIMEOUT( "has-timeout" );
-const algorithm_capabilities::capability_name_t video_input::IS_SEEKABLE( "is-seekable" );
+const algorithm_capabilities::capability_name_t
+video_input::HAS_EOV( "has-eov" );
+const algorithm_capabilities::capability_name_t
+video_input::HAS_FRAME_NUMBERS(
+  "has-frame-numbers" );
```

Comment:
Hmm, it is odd that some of these are one line and some are two. I think with the type always on its own line, the remainder should always fit on a second line?

Reply:
I agree.  I've fixed these.